modules/monitoring-silencing-alerts.adoc

```diff
@@ -10,7 +10,10 @@ You can either silence a specific alert or silence alerts that match a specifica
 
 .Prerequisites
 
-* You have access to the cluster as a developer or as a user with `edit` permissions for the project that you are viewing metrics for.
+* You are a cluster administrator and have access to the cluster as a user with the `cluster-admin` role.
+* You are a non-administator user and have access to the cluster as a user with the following user roles:
+** The `cluster-monitoring-view` role, which allows you to access Alertmanager.
```

Comment:
```suggestion
** The `cluster-monitoring-view` cluster role, which allows you to access Alertmanager.
```

Reply:
fixed. I agree that we need a central place in the monitoring docs to list all of the roles and what they allow. I'll write up a separate issue to add this new topic to the docs.